docs/types.rst

```diff
@@ -218,6 +218,15 @@ String Literals are written with double quotes (``"abc"``). As with integer lite
 
 String Literals support escape characters, such as ``\n``, ``\xNN`` and ``\uNNNN``. ``\xNN`` takes a hex value and inserts the appropriate byte, while ``\uNNNN`` takes a Unicode codepoint and inserts an UTF8 sequence.
 
+.. index:: literal, bytes
+
+Hexadecimal Literals
+--------------------
+
+Hexademical Literals are prefixed with the keyword ``hex`` and are enclosed in double quotes (``hex"001122FF"``). Their content must be a hexadecimal start and their value will be the binary representation of those values.
```

Comment:
In that case, it looks good to me! I'll probably submit a pull request later to mention single-quotes in both String Literals and here.